filebeat/module/shorewall/log/_meta/fields.yml

```diff
@@ -0,0 +1,26 @@
+- name: network
+  type: group
+  description: >
+    Shorewal log files
+  fields:
+    - name: in
+      type: keywork
+    - name: out
+      type: keyword
+- name: action
+  type: group
+  description: >
+    Shorewal network log files
+  fields:
+    - name: one
+      type: keywork
```

Comment:
The actions look like keyword indeed, good catch!

Reply:
I'm possibly going to change them...